test/server/index.js

```diff
@@ -46,6 +46,18 @@ try {
     passwordlessSupported = false;
 }
 
+let thirdPartyPasswordlessSupported;
+let ThirdPartyPasswordlessRaw;
+let ThirdPartyPasswordless;
+
+try {
+    ThirdPartyPasswordlessRaw = require("supertokens-node/lib/build/recipe/thirdpartypasswordless/recipe").default;
+    ThirdPartyPasswordless = require("supertokens-node/recipe/thirdpartypasswordless");
+    thirdPartyPasswordlessSupported = true;
+} catch (ex) {
+    thirdPartyPasswordlessSupported = false;
```

Comment:
You should also change the `/test/featureFlags` API in this file to use this boolean?